<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rigTools</class>
 <widget class="QDialog" name="rigTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rigTools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="loadSelBTN">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="shortNamesCHK">
       <property name="text">
        <string>Show short names</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="selOrderCHK">
       <property name="text">
        <string>Show selection order</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>icon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>node</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>test</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <item>
       <property name="text">
        <string>test attr</string>
       </property>
      </item>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Rig Tagging</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>Create / Tag IK Chains</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="makeIkChainBTN">
                <property name="text">
                 <string>MAKE IK CHAIN</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="solverCMB">
                <item>
                 <property name="text">
                  <string>ikRPsolver</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ikSCsolver </string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ikSplineSolver</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="pvCHK">
              <property name="text">
               <string>poleVector</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>Create / Tag Constraints</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="pointBTN">
                <property name="text">
                 <string>POINT</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="orientBTN">
                <property name="text">
                 <string>ORIENT</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="aimBTN">
                <property name="text">
                 <string>AIM</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="parentBTN">
                <property name="text">
                 <string>PARENT</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="moCHK">
              <property name="text">
               <string>Maintain Offset</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>185</width>
             <height>50</height>
            </size>
           </property>
           <property name="title">
            <string>Misc Tags</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="parentRelBTN">
              <property name="text">
               <string>PARENT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="controllerBTN">
              <property name="text">
               <string>CONTROLLER</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="title">
            <string>Options</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="onlyAttrsCHK">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Only Create Attrs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clearAttrsBTN">
                <property name="text">
                 <string>Clear Attrs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="clearFromCMB">
                <item>
                 <property name="text">
                  <string>From Selected</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>From Skeleton</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tag Skeletons</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>201</width>
         <height>114</height>
        </rect>
       </property>
       <property name="title">
        <string>SkelLink</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Find SkelLinks</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Link Skeletons</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="linkJointsBTN">
              <property name="text">
               <string>Link Joints</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Connect/Disconnect Skeletons</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>215</x>
         <y>5</y>
         <width>182</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>RigParts</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="selRigPartsBTN">
          <property name="text">
           <string>Select RigParts in Scene</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delRigPartsSkelBTN">
          <property name="text">
           <string>Delete RigParts in Skeleton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delRigPartsSceneBTN">
          <property name="text">
           <string>Delete RigParts in Scene</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Misc Utils</string>
      </attribute>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>196</width>
         <height>86</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>LOCK</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_7">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>ROT:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_8">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TRANS:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_6">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>tabWidget</zorder>
  <zorder>tree</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
